src/lib/annotations/Annotator.js

```diff
@@ -319,11 +321,14 @@ class Annotator extends EventEmitter {
 
             if (buttonEl) {
                 buttonEl.classList.add(CLASS_ACTIVE);
-                buttonEl.querySelector(SELECTOR_ANNOTATION_BUTTON_DRAW_ENTER).classList.add(CLASS_HIDDEN);
-                buttonEl.querySelector(SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL).classList.remove(CLASS_HIDDEN);
-                postButtonEl.classList.remove(CLASS_HIDDEN);
+                const enterButtonEl = buttonEl.querySelector(SELECTOR_ANNOTATION_BUTTON_DRAW_ENTER);
+                const cancelButtonEl = buttonEl.querySelector(SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL);
+                annotatorUtil.hideElement(enterButtonEl);
+                annotatorUtil.showElement(cancelButtonEl);
             }
 
+            annotatorUtil.showElement(postButtonEl);
```

Comment:
I will be doing:

```
annotatorUtil.showElement(undoButtonEl);
annotatorUtil.showElement(redoButtonEl);
```

and would like to do it safely (do nothing if !buttonEl || !buttonEl.classList)